packages/fhir-converter/src/templates/cda/Sections/Problems.hbs

```diff
@@ -15,4 +15,22 @@
             {{/each}}
         {{/each}}
     {{/with}}
-{{/if}} 
\ No newline at end of file
+{{/if}}
+{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.20')}}
```

Comment:
Large section we were missing called "Past Medical History" which is really a problems section. Needs its on block though since its down a different xml path.